scripts/check-changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+shopt -s nullglob
+
+root=$1
+prNumber=$2
+
+nonUserFacingString='this is not a user-facing change'
```

Comment:
Think we should have a PR template to make it easier to un-comment this string?

Reply:
Yeah there's definitely something to improve here, though it's not trivial, because this doesn't take into account whether the string occurs in a comment or not. So we'd have to start using some parser.

Maybe it could be a checkbox that has to be ticked instead, something like:

- [ ] This change is **not user-facing** and therefore doesn't need a changelog entry

I'm thinking this would work pretty well